lib/ace/ext/papeeria/latex_behaviour.coffee

```diff
@@ -0,0 +1,187 @@
+define((require, exports, module) ->
+  Behaviour = require("ace/mode/behaviour").Behaviour
+
+  isCommentToken = (token) -> token? and /comment/.test(token.type)
```

Comment:
I thought the `/comment/` regex would be created repeatedly on every invocation. Apparently JS is smart enough to figure it out (in case of regex literals), so feel free to leave it as is